<ui version="4.0" >
 <class>KopeteGVIPropsWidget</class>
 <widget class="QWidget" name="KopeteGVIPropsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>253</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>&amp;General</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lblDisplayName" >
           <property name="text" >
            <string>&amp;Name:</string>
           </property>
           <property name="buddy" >
            <cstring>edtDisplayName</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtDisplayName" />
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="grpIcons" >
         <property name="title" >
          <string>Icons</string>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>9</number>
          </property>
          <property name="topMargin" >
           <number>9</number>
          </property>
          <property name="rightMargin" >
           <number>9</number>
          </property>
          <property name="bottomMargin" >
           <number>9</number>
          </property>
          <property name="horizontalSpacing" >
           <number>6</number>
          </property>
          <property name="verticalSpacing" >
           <number>6</number>
          </property>
          <item row="3" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="2" >
           <widget class="QCheckBox" name="chkUseCustomIcons" >
            <property name="text" >
             <string>Use custom &amp;icons</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="lblOpen" >
            <property name="text" >
             <string>O&amp;pen:</string>
            </property>
            <property name="buddy" >
             <cstring>icnbOpen</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="KIconButton" name="icnbOpen" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="lblClosed" >
            <property name="text" >
             <string>C&amp;losed:</string>
            </property>
            <property name="buddy" >
             <cstring>icnbClosed</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="KIconButton" name="icnbClosed" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>edtDisplayName</tabstop>
  <tabstop>chkUseCustomIcons</tabstop>
  <tabstop>icnbOpen</tabstop>
  <tabstop>icnbClosed</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
